plasmapy/physics/transport/collisions.py

```diff
@@ -592,6 +615,14 @@ def impact_parameter(T,
         bmin = (lambdaBroglie ** 2 + bPerp ** 2) ** (1 / 2)
     else:
         raise ValueError(f"Method {method} not found!")
+    # ARRAY NOTES
+    # it could be that bmin and bmax have different sizes. If Te is a scalar,
+    # T and V will be scalar from _boilerplate, so bmin will scalar.  However
+    # if n_e is an array, than bmax will be an array. if this is the case,
+    # do we want to extend the scalar bmin to equal the length of bmax?
+    # Eh, maybe we should just leave it as a scalar?
```

Comment:
This is a good question that I think deserves some further thought... @namurphy @lemmatum what do you think?

Reply:
My thought on leaving it scalar at the moment was that, well firstly I guess they're likely to be numpy-operator'ed at each other anyway as log( bmax / bmin) in the coulomb log, but philosophically it would be a hands-off approach, letting the user feel like they're just using numpy and if they get scalars or vectors, it's true to their own inputs.

Reply:
This is a tricky one... having the same dimensions would make it easier to maintain the code, but there may be valid reasons for wanting multiple `bmin`s and `bmax`s.
If you were to leave it as is, how would you handle the use of `impact_parameter` within `Coulomb_logarithm`?